OpenIGTLinkIF/MRML/vtkMRMLIGTLConnectorNode.cxx

```diff
@@ -97,6 +97,13 @@ class vtkMRMLIGTLConnectorNode::vtkInternal
   typedef std::map<std::string, vtkSmartPointer <igtlioDevice> > MessageDeviceMapType;
   typedef std::map<std::string, std::vector<std::string> > DeviceTypeToNodeTagMapType;
 
+  struct WasModifyingNode
+  {
+    int Modifying;
+    vtkSmartPointer<vtkMRMLNode> Node;
+  };
+  std::deque<WasModifyingNode> WasModifyingDeque;
```

Comment:
It would be better to include "node" in the name; having the specific container type in the name may not be necessary. So, a name like PendingNodeModifications would be better.

Also, describe in 1-2 sentences why this mechanism is needed at all (otherwise future developers may think it is unnecessary and remove it). It would be even nicer to add a test for it, but probably it is not worth the effort, as it may be difficult to test this.